server/services/store/mattermostauthlayer/mattermostauthlayer.go

```diff
@@ -295,6 +295,30 @@ func (s *MattermostAuthLayer) GetUsersByTeam(teamID string) ([]*model.User, erro
 	return users, nil
 }
 
+func (s *MattermostAuthLayer) GetUsersList(userIDs []string) ([]*model.User, error) {
+	query := s.getQueryBuilder().
+		Select("u.id", "u.username", "u.email", "u.nickname", "u.firstname", "u.lastname", "u.props", "u.CreateAt as create_at", "u.UpdateAt as update_at",
+			"u.DeleteAt as delete_at", "b.UserId IS NOT NULL AS is_bot").
+		From("Users as u").
+		LeftJoin("Bots b ON ( b.UserId = Users.ID )").
+		Where(sq.Eq{"u.deleteAt": 0}).
+		Where(sq.NotEq{"u.roles": "system_guest"}).
+		Where(sq.Eq{"u.id": userIDs})
```

Comment:
Since we are pulling via UserId, we don't need these restrictions, return the users for the Ids that we asked for.

```suggestion
		Select("u.id", "u.username", "u.email", "u.nickname", "u.firstname", "u.lastname", "u.props", "u.CreateAt as create_at", "u.UpdateAt as update_at",
			"u.DeleteAt as delete_at", "b.UserId IS NOT NULL AS is_bot").
		From("Users as u").
		LeftJoin("Bots b ON ( b.UserId = Users.ID )").
		Where(sq.Eq{"u.id": userIDs})
```

Reply:
Sure, I will make these changes and push them.